### ISSUE TITLE
Allow google plus codes to be shared directly into the app

### ISSUE THREAD

**Issue by bb0x88** [organicmaps/organicmaps#9239]
Organic maps is an amazing app, the lonly thing lacking is the points of interest are not as up to date as google maps.

Organic maps has support for plus codes(but not full codes with the city as displayed in google maps currently) - ideally this would be supported

Being able to lookup a point of interest on google maps, share the plus code into the organic maps app, the user gets the best of both worlds - full google point of interest search/database with organic maps taking the user to said point of interest.

Workflow I imagine:
1. Open google maps and search for destination
2. Tap plus code and share
3. select organic maps as sharing desintation
4. Organic maps creates a route from my current location to the destination by importing the plus code

Perhaps taking a normal google maps share link and extracting the plus code to the get the point of interest would also be an even easier/better way for non technical types.

**Comment by lriesebos** [comment]
I agree that this is one of these features where it would make it easier to switch from google maps to organic maps. Reality is that google maps is sometimes just better at finding certain locations. If through a share feature (plus code, share button in google maps, or coordinates) a location could be immediately opened in organic maps, the latter would be more accessible as an alternative to google maps.

**Comment by RedAuburn** [comment]
had a quick look - the intent is basically just a Google Maps link, but the type that needs to be unshortened (https://maps.app.goo.gl/9ivZQ71CKMqRPDtQA). will need an unshortener service as described in https://github.com/organicmaps/organicmaps/issues/475

it doesn
